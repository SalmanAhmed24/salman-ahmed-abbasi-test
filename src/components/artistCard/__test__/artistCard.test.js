import React from 'react';
import ArtistCard from '../artistCard';
import "@testing-library/jest-dom/extend-expect";
import renderer from 'react-test-renderer';

test('<Artistcard> Component test',()=>{
    const component = renderer.create(<ArtistCard thumb_url='../../assets/logo.png' name='salman' facebook='https://facebook.com'/>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

})

