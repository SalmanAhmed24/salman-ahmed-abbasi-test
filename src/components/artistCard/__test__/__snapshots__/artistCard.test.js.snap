// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Artistcard> Component test 1`] = `
<div
  className="main-card"
>
  <div
    className="img-wrap"
  >
    <img
      alt=""
      className="img-card"
      src="../../assets/logo.png"
    />
  </div>
  <div
    className="card-info-wrap"
  >
    <div
      className="card-name"
    >
      <p>
        salman
      </p>
    </div>
    <div
      className="social-link-wrap"
    >
      <p>
        Facebook: 
        <span>
          https://facebook.com
        </span>
      </p>
    </div>
  </div>
</div>
`;
